catalog/files/impl/src/main/java/org/projectnessie/catalog/files/adls/AdlsProgrammaticOptions.java

```diff
@@ -15,117 +15,57 @@
  */
 package org.projectnessie.catalog.files.adls;
 
-import com.google.errorprone.annotations.CanIgnoreReturnValue;
-import java.time.Duration;
+import java.util.LinkedHashMap;
 import java.util.Map;
 import java.util.Optional;
-import java.util.OptionalInt;
-import java.util.OptionalLong;
 import org.immutables.value.Value;
-import org.projectnessie.catalog.secrets.BasicCredentials;
-import org.projectnessie.catalog.secrets.KeySecret;
+import org.projectnessie.catalog.secrets.SecretsProvider;
 
 @Value.Immutable
-public interface AdlsProgrammaticOptions extends AdlsOptions<AdlsFileSystemOptions> {
+@SuppressWarnings("immutables:from")
+public interface AdlsProgrammaticOptions extends AdlsOptions {
```

Comment:
WDYT of renaming "programmatic" to "normalized", e.g. `AdlsNormalizedOptions`? Imho that captures better their current purpose.